src/libs/runtime-utils.js

```diff
@@ -7,6 +7,10 @@ export const DEFAULT_DISK_SIZE = 50
 
 export const usableStatuses = ['Updating', 'Running']
 
+export const defaultDataprocMachineType = 'n1-standard-2'
+export const defaultGceMachineType = 'n1-standard-1'
+export const getDefaultMachineType = isDataproc => isDataproc ? defaultDataprocMachineType : defaultGceMachineType
```

Comment:
could you just stick this ternary in place rather than extracting to a function since it's only used once?

Reply:
By "used once", did you mean "used in only one place"? It is used in two places actually ([here](https://github.com/DataBiosphere/terra-ui/blob/b93bddce8438c87f79c13520878e64d5f67903ce/src/components/NewRuntimeModal.js#L577) and [here](https://github.com/DataBiosphere/terra-ui/blob/b93bddce8438c87f79c13520878e64d5f67903ce/src/libs/runtime-utils.js#L22)). Would that change your comment? 

I'm not attached to factoring it out but it's just that the caller sites were long as they were so I didn't want to make them more difficult to follow.